packages/main/src/TableHeaderCell.ts

```diff
@@ -78,13 +78,20 @@ class TableHeaderCell extends TableCellBase {
 
 	protected ariaRole: string = "columnheader";
 	_popinWidth: number = 0;
+	_individualSlot?: string;
```

Comment:
do we really need this?

Reply:
If I remove this I get warnings.
`Property '_individualSlot' does not exist on type 'TableHeaderCell'.`

See https://github.com/SAP/ui5-webcomponents/pull/9639#discussion_r1728524824 for more information.
@DonkeyCo